onnxruntime/core/framework/sequential_executor.cc

```diff
@@ -306,7 +307,7 @@
 #endif
 
 #ifdef DEBUG_NODE_INPUTS_OUTPUTS
-    session_scope_.dump_context_.program_counter = program_counter_++;
+    session_scope_.dump_context_.program_counter++;
```

Comment:
>program_counter [](http://example.com/codeflow?start=33&length=15)

we might still have a race condition here under multi-stream case. Imagine the case we have two kernels A and B run in parallel - A may hit the line first to have program_counter increased to 1, but before A run to line of 311, B may breach in on line 310 to increase program_counter to 2 ... so when A hit 311 the progrem_counter may already be updated to something unlike it is in the past.